package.json

```diff
@@ -16,11 +16,11 @@
       "eslint --fix",
       "prettier --write",
       "jest --passWithNoTests",
-      "yarn cspell --gitignore"
+      "cspell lint . --gitignore --no-progress --no-must-find-files"
```

Comment:
![image](https://github.com/graphql/graphiql/assets/7361780/042c9e8e-ac56-4921-945c-489fee542280)

you don't need 
- `lint`
- `gitignore` already specified in cspell config
- `.` already specified in https://github.com/graphql/graphiql/blob/2f15e202b194387a1bd78fc778268fb9b8c0fa12/cspell.json#L17

why not just call `yarn lint-cspell`?

Reply:
oops, i meant to remove the `.`.  but yes you are correct, the the commands are the same. are you aware that husky passes only the files that changed to the command as stdin?